extensions/hp-extension/src/index.tsx

```diff
@@ -13,6 +14,17 @@ export default {
    */
   id,
 
+  /**
+   * Register the hanging protocol handlers
+   * @param param0 
+   */
+  preRegistration: ({
+    servicesManager,
+  }) => {
+    const { HangingProtocolService } = servicesManager.services;
+    HangingProtocolService.addCustomViewportSetting('initialScale', "Set initial scaling", initialScaling);
```

Comment:
Register the custom viewport handler - more handlers will be added to define things like preserving previous display set values.